tests/e2e/regression/layernorm.mlir

```diff
@@ -21,7 +21,7 @@
 //   return
 // }
 
-func.func private @layernorm() {
+func.func @layernorm() {
```

Comment:
Looks like this is failing on Android Vulkan, Pixel 6

```
The following tests FAILED:
	 75 - iree/tests/e2e/regression/check_regression_vulkan-spirv_layernorm.mlir_on_android_device (Failed)
```

https://buildkite.com/iree/iree-test-android/builds

Reply:
The failing test is `layernorm_dynamic()`. The static shape test passed. Lei is looking into it.

Reply:
Looking. But the fact that it only fails on Pixel 6 but none of the other GPU targets makes me suspect this is a driver bug (again)..